packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -634,4 +724,23 @@ - (void)setMixWithOthers:(FLTMixWithOthersMessage *)input
   }
 }
 
+- (void)preparePictureInPicture:(FLTPreparePictureInPictureMessage *)input
+                          error:(FlutterError **)error {
+  FLTVideoPlayer *player = self.playersByTextureId[input.textureId];
+  [player usePlayerLayer:CGRectMake(input.left.floatValue, input.top.floatValue,
+                                    input.width.floatValue, input.height.floatValue)
+      canStartPictureInPictureAutomaticallyFromInline:
+          input.enableStartPictureInPictureAutomaticallyFromInline.intValue == 1];
+}
+
+- (void)setPictureInPicture:(FLTPictureInPictureMessage *)input error:(FlutterError **)error {
+  FLTVideoPlayer *player = self.playersByTextureId[input.textureId];
+  [player setPictureInPicture:input.enabled.intValue == 1];
+}
+
+- (nullable NSNumber *)isPictureInPictureSupported:
+    (FlutterError *_Nullable __autoreleasing *_Nonnull)error {
```

Comment:
i believe `__autoreleasing` is the default? also why are you passing in `error` which is not used at all here?

Reply:
The error is generated by pigeon.